components/AddTimer.js

```diff
@@ -8,21 +8,57 @@ class AddTimer extends React.Component {
     this.state = {
       name: '',
       time: '',
-      window: ''
+      window: '',
```

Comment:
What does this line do?

Reply:
ya'know... I'm not 100% on that. I'll look at it and get back to you on that.

Reply:
Okay, looked into it. It's so we can have state for the 'window' just like the 'time' and 'name' above. I think the comma is for the next pieces of state that should be in reference to the placeholders.